### ISSUE TITLE
Configurable Product Bundle doesn't load properly in Order Creation

### ISSUE THREAD

**Issue by malinajirka** [woocommerce/woocommerce-android#10930]
**Describe the bug**
When I add a configurable product bundle into an order during the Order Creation flow, the sub-items never finish loading.

**To Reproduce**
1. Go to Orders
2. Click on Create Order
3. Click on Add Product
4. Select a product Bundle
5. Add it to the order
6. Notice the products within the bundle never finish loading.

**Screenshots**
![Screenshot 2024-02-23 at 20 46 04](https://github.com/woocommerce/woocommerce-android/assets/2261188/f0edaf11-d3ef-486b-8859-a7612823ef77)

**Expected behavior**
All the products within the bundle load correctly. 

Tablet, Android 33, Emulator

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by malinajirka** [comment]
@atorresveiga Is this issue on your radar?

**Comment by atorresveiga** [comment]
@ThomazFB, did you work on this issue with the bundle configuration, or was it a different one? 🤔

**Comment by ThomazFB** [comment]
Indeed, I fixed a never-ending loading for the bundle's feature. I noticed the problem when fixing another issue:
- https://github.com/woocommerce/woocommerce-android/issues/11761 | https://github.com/woocommerce/woocommerce-android/pull/12391

This additional loading problem was fixed on https://github.com/woocommerce/woocommerce-android/pull/12683. I think at the time, I didn't notice there was an issue tracking the problem already and didn't tag it here.
